raiden/transfer/unlock.py

```diff
@@ -0,0 +1,141 @@
+from typing import Optional
+
+from eth_utils import to_checksum_address, to_hex
+
+from raiden.constants import EMPTY_HASH
+from raiden.exceptions import RaidenUnrecoverableError
+from raiden.storage.restore import channel_state_until_state_change
+from raiden.transfer.channel import get_batch_unlock_gain
+from raiden.transfer.identifiers import CanonicalIdentifier
+from raiden.transfer.state import ChainState, NettingChannelState, NettingChannelEndState
+from raiden.transfer.utils import get_state_change_with_balance_proof_by_locksroot, \
+    get_event_with_balance_proof_by_locksroot
+from raiden.transfer.views import get_channelstate_by_token_network_and_partner
+from raiden.utils import Address
+from raiden.utils.typing import TokenNetworkID
+
+def get_channel_state(
```

Comment:
There are already some functions to retrieve chanenl state on views, why is this function needed?

Reply:
This function was part of handle_batch_unlock function. I just encapsulated it into another function in order to make it shorter and cleaner in the light version, without changing the idea behind the code. I like too keep the functions as shorter as possible, grouping them in smaller functions if necessary.

Anyway, if there is a function that do exactly this (getting a channel_state or 'die' trying) I'll cyhange it by it.

If there's a method